mysql-test/mroonga/storage/column/time/fractional_seconds/t/null.test

```diff
@@ -0,0 +1,42 @@
+# -*- mode: sql; sql-product: mysql -*-
+#
+# Copyright (C) 2024  Kodama Takuya <otegami@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+--source ../../../../../include/mroonga/have_mroonga.inc
```

Comment:
```suggestion
--source ../../../../../include/mroonga/have_fractional_seconds.inc
--source ../../../../../include/mroonga/have_mroonga.inc
```

Reply:
Hmm. It seems that all supported MySQL/MariaDB supports fractional seconds.
So we don't need to use `time/fractional_seconds/`.

Reply:
Hmm. `time/{fractional_seconds/,}t/with_index.test` is better than `time/with_index/t/{with,without}_fractional_seconds.test`. Let's use keep this path.

```console
$ find mysql-test/mroonga/storage/column/time
mysql-test/mroonga/storage/column/time
mysql-test/mroonga/storage/column/time/t
mysql-test/mroonga/storage/column/time/t/with_index.test
mysql-test/mroonga/storage/column/time/r
mysql-test/mroonga/storage/column/time/r/with_index.result
mysql-test/mroonga/storage/column/time/fractional_seconds
mysql-test/mroonga/storage/column/time/fractional_seconds/t
mysql-test/mroonga/storage/column/time/fractional_seconds/t/with_index.test
mysql-test/mroonga/storage/column/time/fractional_seconds/r
mysql-test/mroonga/storage/column/time/fractional_seconds/r/with_index.result
```

Reply:
fix: [8fc34c3](https://github.com/mroonga/mroonga/pull/791/commits/8fc34c371426e4f2fee916f4076d5a1b0d75ba86)
Ahh Isee. Could we remove the `MRN_HAVE_MYSQL_TYPE_TIME2` flag from the source code now?
If so,  I will remove it in another PR.
```cpp
#ifdef MRN_HAVE_MYSQL_TYPE_TIME2
  case MYSQL_TYPE_TIME2:
    error = generic_store_bulk_time2(field, buf);
    break;
#endif
```

Reply:
Yes.